modules/sqs/src/main/scala/vinyldns/sqs/queue/SqsMessage.scala

```diff
@@ -0,0 +1,38 @@
+package vinyldns.sqs.queue
+import java.util.Base64
+
+import cats.implicits._
+import com.amazonaws.services.sqs.model.Message
+import vinyldns.core.domain.ZoneCommand
+import vinyldns.core.protobuf.ProtobufConversions
+import vinyldns.core.queue.CommandMessage
+import vinyldns.proto.VinylDNSProto
+import vinyldns.sqs.queue.SqsMessageQueue.EmptySqsMessageContents
+import vinyldns.sqs.queue.SqsMessageType.{SqsRecordSetChangeMessage, SqsZoneChangeMessage}
+
+final case class SqsMessage(receiptHandle: String, command: ZoneCommand) extends CommandMessage
```

Comment:
pulled this out to a separate file.  Moved functions related to the SqsMessge here